Formula/p/pyre.rb

```diff
@@ -0,0 +1,129 @@
+class Pyre < Formula
+  include Language::Python::Virtualenv
+
+  desc "Performant type-checking for python"
+  homepage "https://pyre-check.org"
+  url "https://files.pythonhosted.org/packages/fd/94/bbbb7b51310bc17fd80453f17c72fdf5079dbfe97558c511295be50a4880/pyre-check-0.9.22.tar.gz"
+  sha256 "e082f926dff71661959535c3936fca5ad40a44858b5fd3e99009a616a1b57083"
+  license "MIT"
+  head "https://github.com/facebook/pyre-check.git", branch: "main"
+
+  depends_on "rust" => :build
+  depends_on "libyaml"
+  depends_on "python@3.12"
```

Comment:
can we also do py3.13?
```suggestion
  depends_on "python@3.13"
```

Reply:
We should test it, was a little irritated about the [tox](https://github.com/facebook/pyre-check/blob/fa94e180b696f8da3fe564fc67addf5b3e649f0c/pyproject.toml#L6-L8) classifier


```config
[tox]
envlist = py38, py39, py310, py311, py312, lint
```

but [setup.py](https://github.com/facebook/pyre-check/blob/fa94e180b696f8da3fe564fc67addf5b3e649f0c/scripts/pypi/setup.py#L99-L112) seems to be a little more off ...

```py
       classifiers=[
            "Development Status :: 5 - Production/Stable",
            "Environment :: Console",
            "Intended Audience :: Developers",
            "License :: OSI Approved :: MIT License",
            "Operating System :: MacOS",
            "Operating System :: POSIX :: Linux",
            "Programming Language :: Python",
            "Programming Language :: Python :: 3",
            "Programming Language :: Python :: 3.8",
            "Programming Language :: Python :: 3.9",
            "Topic :: Software Development",
            "Typing :: Typed",
        ],
```